src/components/MultiStepIndicator/MultiStepIndicator.tsx

```diff
@@ -12,32 +9,58 @@ import { SIZES, MULTI_STEP_TYPES, STEP_STATUSES, TEXT_PLACEMENTS } from "./Multi
 import { getTestId } from "../../tests/test-ids-utils";
 import { ComponentDefaultTestId } from "../../tests/constants";
 import styles from "./MultiStepIndicator.module.scss";
+import { withStaticProps, VibeComponentProps, VibeComponent, SubIcon } from "../../types";
+import { IconType } from "../Icon/IconConstants";
 
-const MultiStepIndicator = forwardRef(
+export type Step = {
+  titleText: string;
+  subtitleText: string;
+  status: STEP_STATUSES;
+};
+
+interface MultiStepIndicatorProps extends VibeComponentProps {
+  steps: Step[];
+  type?: MULTI_STEP_TYPES;
+  stepComponentClassName?: string;
+  dividerComponentClassName?: string;
+  fulfilledStepIcon?: SubIcon;
+  fulfilledStepIconType?: IconType.SVG | IconType.ICON_FONT;
+  isFulfilledStepDisplayNumber?: boolean;
+  onClick?: (stepNumber: number) => void;
+  textPlacement?: TEXT_PLACEMENTS;
+  size?: SIZES;
+}
+
+const MultiStepIndicator: VibeComponent<MultiStepIndicatorProps> & {
+  types?: typeof MULTI_STEP_TYPES;
```

Comment:
@SergeyRoyt I think that we can remove the ? after you merged the new use static props hook, am I right?

Reply:
@hadasfa can you elaborate on that please?